tests/basic/upgrade_cluster_test.go

```diff
@@ -90,6 +91,7 @@ var _ = Describe("{UpgradeCluster}", func() {
 
 				err = Inst().S.UpgradeScheduler(version)
 				if err != nil {
+					err = Inst().V.RefreshDriverEndpoints()
```

Comment:
can we add refershnode registry also?
I see only anthos is doing it but if we add it here it will common for all the platforms

Reply:
sure, done !!